### PR TITLE
Add Cyprus

### PR DESCRIPTION
**Pull request by gregn610**
refs #

For information, read and make sure you're okay with the [CONTRIBUTING document](https://github.com/novafloss/workalendar/blob/master/CONTRIBUTING.rst#adding-new-calendars).

- [x] Tests with a significant number of years to be tested for your calendar.
- [x] Docstrings for the Calendar class and specific methods.
- [x] Calendar country / label added to the README.rst file,
- [x] Changelog amended with a mention like: "Added ``<country>`` by ``@pseudo`` (#)"

- [ ] Tests with a significant number of years to be tested for your calendar.
- [x] Changelog amended with a mention describing your changes.

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+1.0.1 (2017-01-07)
+------------------
+
+- Add Cyprus. thx @gregn610 (#).
```
Comment (Natim) [inline CHANGELOG]: #174

File: workalendar/tests/test_europe.py [hunk 2]
```diff
@@ -98,6 +99,28 @@ def test_year_2016(self):
         self.assertIn(date(2016, 12, 26), holidays)  # St. Stephen's Day Prvi d
 
 
+class Cyprus(GenericCalendarTest):
+    cal_class = Cyprus
+
+    def test_year_2017(self):
+        holidays = self.cal.holidays_set(2017)
+        self.assertIn(date(2017, 1, 1), holidays)    # New Year's Day
```
Comment (Natim) [inline workalendar/tests/test_europe.py]: Can we update the test to make sure no other days are present in the set? Testing the length might be enough.

File: CHANGELOG [hunk 3]
```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+1.0.1 (2017-01-07)
```
Comment (brunobord) [inline CHANGELOG]: I'd rather have this:

```
master (unreleased)
-------------------
```

you never know when the next release would happen ;o)

File: workalendar/tests/test_europe.py (hunk 2 above)
Comment (brunobord) [inline workalendar/tests/test_europe.py]: It's not blocking IMHO. When I can get some time, I'll have to change every test to check against a set(), that'll make sure there are no extra or missing days

**Pull request merged**